enos/enos-scenario-smoke.hcl

```diff
@@ -296,6 +296,22 @@ scenario "smoke" {
     }
   }
 
+  step "verify_bruteforcing" {
+    skip_step  = semverconstraint(var.vault_product_version, "<1.13.0-0")
+    module     = module.vault_verify_bruteforcing
+    depends_on = [step.verify_read_test_data]
```

Comment:
Can we get a comment here that explains that we're depending on verify_read_test_data because it creates a user that we also want to use for this verification?